### ISSUE TITLE
about Bidirectional IPv6 Connectivity and DNS-Based Service Discovery

### ISSUE THREAD

**Issue by lancersky** [openthread/openthread#8861]
**Describe the bug** A clear and concise description of what the bug is.
As shown below, I need to specify the `backboneIfName` to `wlan0` or `eth0`, Therefore, the PC can access the Thread network.
But if OTBR also is a WiFi router, and the PC connects to OTBR directly. Can the PC access the Thread network?
Is there any modification required? Change `backboneIfName` to `br-lan`?
![image](https://user-images.githubusercontent.com/11283852/224872197-a9d50164-3bc4-4bd3-b5bb-d283b11c32d7.png)

**Comment by wgtdkp** [comment]
Yes, set the `backboneIfName` to the wlan interface should work.
